src/schema/me/create_bidder_mutation.js

```diff
@@ -0,0 +1,30 @@
+import { GraphQLNonNull, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import Bidder from "schema/bidder"
+
+export default mutationWithClientMutationId({
+  name: "Bidder",
```

Comment:
#minor probably should be `CreateBidder` (so we get the types `CreateBidderInput`, `CreateBidderPayload`, etc.)